src/components/packet/packet.tsx

```diff
@@ -39,83 +41,47 @@ export function Packet() {
         }))
     }
 
-    async function fetchLeaderboardInfo(link: string) {
```

Comment:
Not sure why this got removed, will add it back in. If we do this on the success/empty screen itself, it'd just spawn in after being rendered, thats why it should be done here.